### PR TITLE
[Android] Warnings shouldn't be suppressed

### PR DESCRIPTION
**Pull request by adamradocz**
**Description of the Change**
Warning messages shouldn't be suppressed but resolved.
The new function uses API level 13, we target the minimum 19.

**Release Notes**
N/A

### CODE REVIEW

**Comment by adamradocz** [comment]
Deprecated function call resolved.

**Comment by adamradocz** [comment]
@JuhaSointusalo I prepended the branch, next time I'll prepend the commit message too.

File: android/BOINC/app/src/main/java/edu/berkeley/boinc/ProjectDetailsFragment.java [hunk 1]
```diff
@@ -112,9 +113,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 	@Override
 	public void onAttach(Activity activity) {
 		// first time fragment can get a valid context (before this, getActivity() will return null!)
-		Display display = activity.getWindowManager().getDefaultDisplay(); 
-		width = display.getWidth();
-		height = display.getHeight();
+		Display display = activity.getWindowManager().getDefaultDisplay();
```
Comment (brevilo) [inline android/BOINC/app/src/main/java/edu/berkeley/boinc/ProjectDetailsFragment.java]: This line didn't change it's functional content so I presume the indentation changed from `tab` to `space` or vice versa? If that's correct, did you ensure the indentation style of the two remaining lines (115, 124 in your version) are identical?

Comment (adamradocz) [reply android/BOINC/app/src/main/java/edu/berkeley/boinc/ProjectDetailsFragment.java]: I checked it's the same.

**Comment by brevilo** [comment]
@AenBleidd anything to add?

**Comment by brevilo** [comment]
Thanks guys!

**Pull request merged**